### ISSUE TITLE
PDF Created In Acrobat omits lines

### ISSUE THREAD

**Issue by slickrory** [aspose-pdf/Aspose.PDF-for-.NET#32]
Creating a PDF via Acrobat does not persist drawn lines in a pdf that is created from a Word Doc. It used to work. This does not happen when creating a PDF via Word or other applications, but only with Acrobat.

**Comment by asadalikhan90** [comment]
@flipcents 

Are you using Aspose.PDF at your end? Can you please share a bit more details of the issue that you are facing while sharing the code snippet and the sample files. Please note that we recommend posting issues related to the API in our [official support forum](https://forum.aspose.com/c/pdf) where you can easily share files and track the progress appropriately against the issue investigation. You can please create a topic there in case you are facing some issues while using Aspose.PDF and we will further assist you accordingly.

**Comment by roryagilethought** [comment]
@asadalikhan90 it was specific to the last upgrade. Same standard code we always used which always worked fine, all we did was update the library. And it only happened with PDF's created in Acrobat. Real easy to test, Create a document with drawn lines in Word, then create a PDF with drawn lines in Acrobat. One will display the lines one will not in Aspose.PDF

With that said, I posted this issue SIX months ago, hahaha... We wrapped that project up back in February. The business didn't want to roll back the library as rolling it back had some issues with thumbnails as they decided this was less of an impact and we were not getting any feedback from this issue. I don't have access to the code anymore.
